Changes

```diff
@@ -127,6 +127,12 @@ Working version
   needed as the pagetable heads towards retirement).
   (David Allsopp, review by Xavier Leroy)
 
+- #9534: Introduce a naked pointers checker mode to the runtime,
+  to print information on relevant values when they are
+  inspected by the garbage collector.
+  (KC Sivaramakrishnan, with contributions by Stephen Dolan and David Allsopp,
```

Comment:
You should add yourself here @Engil. And to be consistent with the other entries in the file, perhaps do:

```
Enguerrand Decorne, KC Sivaramakrishnan, Xavier Leroy, Stephen Dolan, David Allsopp, Nicolás Ojeda Bär, review by ...
```
Xavier contributed the inline assembly and Nicolás kick-started the work on Windows support.